src/App.css

```diff
@@ -1,39 +1,44 @@
-.App {
```

Comment:
- [ ] It would be better if we delete all the boilerplate that got created by the CRA (texts, images, and styles). Here are the hints on which files to delete:
  - Src -: `index.css`, `App.test.js`, `logo.svg`, `reportWebVitals.js`, `setupTest.js`
  - Public -: `favicon.ico`, `logo192.png`, `logo512.png`, `manifest.json`, `robots.txt`